### DISCUSSION TITLE
Client Side Rendering Support

### DISCUSSION THREAD

**Discussion by Just-Moh-it** [remotion-dev/remotion#1038] [Q&A] [answered]
Remotion provides a way to play videos on the client side using @remotion/player. But there's no way I could find to export this video preview into a `.mp4` or any other format without involving a server. Currently, if there would be no way, the user would have to be asked to turn on their screen-recorder and maximize their player to record the @remotion/player element, and this is inconvenient and records at the rate of video playback.

I'm building something like [Animockup](https://animockup.com), and there's **support to export** to `.webm` file (using the CanvasRecorder.js lib) **on the client-side**. Is there something similar for recording or exporting a @remotion/player element contents to a video file, with, or without audio and without involving a server?

**Comment by JonnyBurger** [comment, marked as answer]
Current status and future options will be documented here: https://www.remotion.dev/docs/miscellaneous/render-in-browser
